guides/common/modules/proc_installing-amazon-ec2-plugin.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Installing_Amazon_EC2_Plugin_{context}"]
+= Installing Amazon EC2 Plugin
+
+Install the Amazon EC2 plugin to attach an EC2 compute resource provider to {Project}.
+This allows you to manage and deploy hosts to EC2.
+
+.Procedure
+. Install the EC2 compute resource provider on your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --enable-foreman-compute-ec2
+----
+. Optional: In the {ProjectWebUI}, navigate to *Administer > About* and select the _compute resources_ tab to verify the installation of the Amazon EC2 plugin.
```

Comment:
```suggestion
. Optional: In the {ProjectWebUI}, navigate to *Administer* > *About* and select the *compute resources* tab to verify the installation of the Amazon EC2 plugin.
```
I found that in the repository, it is more common to wrap the name of the tab in \* \*, rather than in \_ \_

Reply:
Applied.